src/region/RegionAnnotations.tsx

```diff
@@ -108,13 +119,13 @@ export default class RegionAnnotations extends React.Component<Props, State> {
             <>
                 {/* Layer 1: Saved annotations */}
                 <svg className="ba-RegionAnnotations-list">
-                    {annotations.map(({ id, target }) => (
+                    {this.sortShapes().map(({ id, target }) => (
```

Comment:
Probably not a big deal, but would there be any benefits of moving this sort out of the render function to avoid sorting each render? Maybe in the redux connected container? Or storing it in state?

Reply:
Sure, it's probably time for the list of existing annotations to move to its own component. I'll move it.